saleor/graphql/core/types/tests/test_money.py

```diff
@@ -0,0 +1,14 @@
+from decimal import Decimal
+
+from prices import Money
+
+from ..money import Money as MoneyObject
+
+
+def test_money_object():
+    money = Money(Decimal("12.950000"), "USD")
+    resolve_info = None
+
+    assert MoneyObject.resolve_amount(money, resolve_info) == Decimal("12.95")
+    assert MoneyObject.resolve_fractional_amount(money, resolve_info) == 1295
+    assert MoneyObject.resolve_fraction_digits(money, resolve_info) == 2
```

Comment:
maybe let's add cases for JPY and other currencies that have non-2 fraction digits